### PR TITLE
[ch3749] Customise meta description for committee

### PR DESCRIPTION
**Pull request by cacahouwete**
https://app.clubhouse.io/enmarche/story/3749/modifier-les-meta-descriptions-des-pages-de-comit%C3%A9s-locaux

### CODE REVIEW

File: templates/committee/_layout.html.twig [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 {% set is_host = is_host(committee) %}
 
+{%- block meta_description -%}
+    {{ committee.name }} est un comité local de La République En Marche ! créé par {{ committee_hosts[0].partialName }} et situé dans la ville de {{ committee.cityName }}. Il est composé de {{ 'committee.members_count'|transchoice(committee.membersCount) }} adhérents actifs. Les comités locaux sont le fondement de notre mouvement, rejoignez-en un pour participer au renouvellement de la politique française !
```
Comment (B-Galati) [inline templates/committee/_layout.html.twig]: Is it possible to add some new lines here ? Or it would not fit the meta_description content ?

**Comment by thomasbisignani** [comment]
IMO you can change also the `og:meta` and `twitter:description` meta tags. It's the same content.

File: templates/committee/_layout.html.twig [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 {% set is_host = is_host(committee) %}
 
+{%- block meta_description -%}
+    {{ committee.name }} est un comité local de La République En Marche !
+    {% if committee_hosts is not empty %}créé par {{ committee_hosts[0].partialName }} et {% endif %}situé dans la ville de {{ committee.cityName }}.
+    Il est composé de {{ 'committee.members_count'|transchoice(committee.membersCount) }} adhérents actifs.
```
Comment (thomasbisignani) [inline templates/committee/_layout.html.twig]: You should remove `adhérents` here. It's already present in the `committee.members_count` translation.

File: translations/messages.fr.yml [hunk 3]
```diff
@@ -74,6 +74,7 @@ committee.message_deleted: Votre message a bien été supprimé.
 committee.message_published: Votre message a bien été publié.
 
 committee.members_count: '1 adhérent|%count% adhérents'
+committee.members_count_actif: '1 adhérent actif|%count% adhérents actifs'
```
Comment (thomasbisignani) [inline translations/messages.fr.yml]: `committee.active_members_count`

**Pull request merged**